.travis.yml

```diff
@@ -7,7 +7,7 @@ stages:
 .doc_test: &doc_test
   stage: "Documentation (Test)"
   os: linux
-  julia: 1.0
+  julia: 1.4
```

Comment:
```suggestion
  julia: 1
```

Reply:
1 would be running it on 1.5, which is not stable, but only RC, or now?

Reply:
This is equivalent to 1.4 currently and won't be updated to 1.5 until sometime after it has officially released. See: https://github.com/travis-ci/travis-build/blob/master/lib/travis/build/script/julia.rb#L185

Reply:
Oh, I see, my bad, I somehow remember it wrongly.

Reply:
No problem at all. It's not very intuitive